include/swift/AST/LayoutConstraint.h

```diff
@@ -108,6 +114,26 @@ class LayoutConstraintInfo : public llvm::FoldingSetNode {
     return isNativeRefCountedObject(Kind);
   }
 
+  bool isClass() const {
+    return isClass(Kind);
+  }
+
+  bool isNativeClass() const {
+    return isNativeClass(Kind);
+  }
+
+  bool isAnyClass() const {
```

Comment:
I still find it confusing that we have `isClass()` and `isAnyClass()`. If someone wants the `isClass()` behavior, they can just check the kind exactly. Are you opposed to giving `isClass`() the behavior that `isAnyClass()` has (i.e., so `isClass()` answers the question "is it some kind of class?") and removing `isAnyClass()` entirely?